README.md

```diff
@@ -72,7 +72,7 @@ This scenario applies if:
 import Ember from 'ember';
 import pagedArray from 'ember-cli-pagination/computed/paged-array';
 
-Ember.ArrayController.extend({
+export default Ember.ArrayController.extend({
```

Comment:
Let's take it a step further and replace `Ember.ArrayController` with `Ember.Controller`?